modules/nf-core/custom/dumpsoftwareversions/custom-dumpsoftwareversions.diff

```diff
@@ -0,0 +1,19 @@
+Changes in module 'nf-core/custom/dumpsoftwareversions'
+--- modules/nf-core/custom/dumpsoftwareversions/main.nf
++++ modules/nf-core/custom/dumpsoftwareversions/main.nf
+@@ -2,10 +2,10 @@
+     label 'process_single'
+ 
+     // Requires `pyyaml` which does not have a dedicated container but is in the MultiQC container
+-    conda "${moduleDir}/environment.yml"
```

Comment:
Wouldn't it be simpler to update the central module and bump it here? The Kallisto modules have the environment yml too.

Reply:
Absolutely. But I thought I would just patch this for now because the plan is to remove this module altogether when moving to nf-test.

Reply:
Can create a small follow up PR before the release bumping the module and re-installing if you like? I'll merge this for now so others can test the fix works.